lib/ansible/plugins/inventory/foreman.py

```diff
@@ -25,12 +25,18 @@
       url:
         description: url to foreman
         default: 'http://localhost:3000'
+        env:
+            - name: FOREMAN_SERVER
```

Comment:
missing `version_added`

Reply:
What about backporting to 2.7? I would like to keep that option open. Can I put 2.7 and commit to putting in the backport?

Reply:
features dont get backported

Reply:
I can added this via:

```diff
diff --git a/lib/ansible/plugins/inventory/foreman.py b/lib/ansible/plugins/inventory/foreman.py
index 6827e558d6..8c6e522fc5 100644
--- a/lib/ansible/plugins/inventory/foreman.py
+++ b/lib/ansible/plugins/inventory/foreman.py
@@ -32,6 +32,7 @@ DOCUMENTATION = '''
         required: True
         env:
             - name: FOREMAN_USER
+        version_added: 2.8
       password:
         description: foreman authentication password
         required: True
```

And that will give

```
= user
        foreman authentication user

        set_via:
          env:
          - name: FOREMAN_USER
        
        version_added: 2.8
```

But this isn't actually correct - the field was not added in 2.8. Only the option to get the field from the environment variable is added in 2.8. The field has been there for a while.

Alternatively

```diff
diff --git a/lib/ansible/plugins/inventory/foreman.py b/lib/ansible/plugins/inventory/foreman.py
index 6827e558d6..cbf64581d8 100644
--- a/lib/ansible/plugins/inventory/foreman.py
+++ b/lib/ansible/plugins/inventory/foreman.py
@@ -32,6 +32,7 @@ DOCUMENTATION = '''
         required: True
         env:
             - name: FOREMAN_USER
+            version_added: 2.8
       password:
         description: foreman authentication password
         required: True
```

does not work, with a formatting error.

Reply:
need to indent to match 'name:' also quote it , needs to be string, not float

Reply:
its the specific entry 'that env var' that was added, not the 'env' section

Reply:
> need to indent to match 'name:'

I can do that, and then `ansible-doc -t inventory foreman`, but it also doesn't show any information in that command about the version.

```
- url
        url to foreman
        [Default: http://localhost:3000]
        set_via:
          env:
          - name: FOREMAN_SERVER
        

= user
        foreman authentication user

        set_via:
          env:
          - name: FOREMAN_USER
```

If that's intended, then it's fine by me.

Reply:
ansible-doc doesnt need it cause its 'current version' that is mostly for the html docsite

Reply:
ok, current state of PR reflects this.

Some other users who have touched this plugin: @milliams @vmpr, other people who touched it when it was still a script @keithresar @giovannisciortino

Just thought I would ping them in case they have any other review comments for this behavior change.